--- ZynAddSubFX-2.2.1/src/UI/Makefile.orig	2007-12-21 13:48:31.153320111 +0100
+++ ZynAddSubFX-2.2.1/src/UI/Makefile	2007-12-21 13:48:38.152252380 +0100
@@ -2,6 +2,8 @@
 
 %.cc  : %.fl
 	fluid -c $<
+%.h  : %.fl
+	fluid -c $<
 
 objects=WidgetPDial.o PresetsUI.o EnvelopeUI.o LFOUI.o FilterUI.o VirKeyboard.o ConfigUI.o\
 	SUBnoteUI.o ResonanceUI.o OscilGenUI.o ADnoteUI.o PADnoteUI.o EffUI.o BankUI.o \
